File: frontend/src/utils/currency.ts
// Курсы валют для PoliceRP
export const EXCHANGE_RATES = {
  AR_TO_BT: 3, // 1 АР = 3 БТ
} as const;

/**
 * Конвертирует АР в БТ по текущему курсу
 */
export function convertArToBt(arAmount: number): number {
  return arAmount * EXCHANGE_RATES.AR_TO_BT;
}

/**
 * Конвертирует БТ в АР по текущему курсу
 */
export function convertBtToAr(btAmount: number): number {
  return Math.floor(btAmount / EXCHANGE_RATES.AR_TO_BT);
}

/**
 * Форматирует сумму в БТ для отображения
 */
export function formatBt(amount: number): string {
  return `${amount} БТ`;
}

/**
 * Проверяет, достаточно ли БТ для оплаты суммы в АР
 */
export function canPayWithBt(btBalance: number, arAmount: number): boolean {
  return btBalance >= convertArToBt(arAmount);
}

File: backend/app/utils/currency.py
"""
Утилиты для работы с валютами PoliceRP
"""

# Курсы валют
EXCHANGE_RATES = {
    "AR_TO_BT": 3,  # 1 АР = 3 БТ
}


def convert_ar_to_bt(ar_amount: int) -> int:
    """Конвертирует АР в БТ по текущему курсу"""
    return ar_amount * EXCHANGE_RATES["AR_TO_BT"]


def convert_bt_to_ar(bt_amount: int) -> int:
    """Конвертирует БТ в АР по текущему курсу"""
    return bt_amount // EXCHANGE_RATES["AR_TO_BT"]


def can_pay_with_bt(bt_balance: int, ar_amount: int) -> bool:
    """Проверяет, достаточно ли БТ для оплаты суммы в АР"""
    return bt_balance >= convert_ar_to_bt(ar_amount)